src/components/ListItemIcon/ListItemIcon.vue

```diff
@@ -189,8 +198,11 @@ export default {
 		},
 
 		cssVars() {
+			// Don't use margin to calculate the height if noMargin
+			const margin = this.noMargin ? 0 : this.margin
```

Comment:
why not take "margin='0'" as property for disabling ? probably we don't want people supplying custom margins I guess ?

Reply:
> probably we don't want people supplying custom margins I guess ?

Yep :/